VSRAD.DebugServer/IPC/Commands.cs

```diff
@@ -13,8 +16,9 @@ public static ICommand ReadCommand(this IPCReader reader)
                 case 0: return Execute.Deserialize(reader);
                 case 1: return FetchMetadata.Deserialize(reader);
                 case 2: return FetchResultRange.Deserialize(reader);
-                case 3: return Deploy.Deserialize(reader);
-                case 4: return ListEnvironmentVariables.Deserialize(reader);
+                case 3: return PutFileCommand.Deserialize(reader);
+                case 4: return Deploy.Deserialize(reader);
+                case 5: return ListEnvironmentVariables.Deserialize(reader);
```

Comment:
It would be better to define byte type enum with command types `enum CommandType : byte { Execute = 0, ..., ListEnvironmentVariables = 5 }`

Reply:
That's a great suggestion, thanks! I'll apply it in subsequent PRs along with command/response internal name changes as indicated by this TODO
```
// TODO: Rename commands and responses to follow a predictable pattern of XCommand -> XResponse
// (e.g. ExecuteCommand and ExecuteResponse instead of Execute and ExecutionFinished)
```